README.cn.md

```
@@ -0,0 +1,260 @@
+/career-ops pipeline       → 处理待办 URL
+/career-ops contacto       → 生成 LinkedIn 外联消息
+/career-ops deep           → 深度公司研究
+/career-ops training       → 评估课程/证书
+/career-ops project        → 评估作品集项目
+```
+
+或者直接粘贴职位 URL 或职位描述，career-ops 会自动识别并运行完整流程。
+
+## 工作原理
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)